src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -381,3 +381,13 @@ export const getHomelessOrAtRisk = formData => {
 
 export const isNotUploadingPrivateMedical = formData =>
   _.get(DATA_PATHS.hasPrivateRecordsToUpload, formData) === false;
+
+export const showPtsdCombateConclusion = form =>
+  form['view:uploadPtsdChoice'] === 'answerQuestions' &&
+  (form['view:selectablePtsdTypes']['view:combatPtsdType'] ||
```

Comment:
just wondering if this is safe to do without first making sure that `form['view:selectablePtsdTypes']` is a valid object. Is it possible to use the `get` utility method instead?

Reply:
It is, and fixed!